### PR TITLE
Fixes #23239 - Migrate two-pane to breadcrumb-bar

### PR DESCRIPTION
**Pull request by sharvit**

### CODE REVIEW

**Comment by dLobatog** [comment]
[test]
[test default]

**Comment by dLobatog** [comment]
The original problem was this, btw

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,8360 @@
+{
```
[the remark this answers is not in the document]
Comment (sharvit) [reply package-lock.json]: Yes it was an accident, let me update it quickly.
We usually `.gitignore` this file but it also not a perfect solution, `npm` actually bealive this file should be apart of you sourcecode like the `Gemfile.lock` (which we also gitignore)

**Comment by sharvit** [comment]
Thanks @dLobatog @iNecas 
Just removed the `package-lock.json` from the commit.

File: package-lock.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (iNecas) [reply package-lock.json]: Could you add that to gitignore, to prevent this situation further or is there any reason not to?

**Comment by adamruzicka** [comment]
For some reason the buttons are wrongly placed in the popup for me. Is this within the scope of this PR?

image

**Comment by sharvit** [comment]
@adamruzicka, what browser are you using?

**Comment by adamruzicka** [comment]
I'm on firefox 59. If I zoom out to 90%, the buttons fall back in line. Given this happens on other pages as well, it is probably out of scope of this and should be fixed elsewhere.

**Comment by iNecas** [comment]
[test]
[test default]

**Comment by iNecas** [comment]
@adamruzicka any other comments or good to merge?

**Comment by adamruzicka** [comment]
Good to go

**Comment by iNecas** [comment]
Thanks @sharvit and @adamruzicka

**Pull request merged**